tests/fixtures.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
-from copy import deepcopy
-
 import pytest
-
+from sys import version_info
 from pydash._compat import iteritems
 
+if version_info[0] > 2:
```

Comment:
Or does that not modify the function spec which wouldn't really test that annotations are handled correctly?